docs/topic_guide/channel_response.rst

```diff
@@ -0,0 +1,146 @@
+It is a useful quantity when performing thermal analysis of imaging data, such as a differential emission measure or filter ratio analysis.
+The temperature response is defined as,
+
+.. math::
+
+    K(T) = \int\mathrm{d}\lambda\,R(\lambda)S(\lambda,T)\quad[\mathrm{DN}\,\mathrm{pixel}^{-1}\,\mathrm{s}^{-1} \,\mathrm{cm}^5]
+
+It has a physical type of DN per pixel per unit time per unit emission measure.
+Note that the temperature response is a function of *both* the instrument properties as well as the atomic physics of the emitting source.
+The temperature response is related to the observed intensity in a given pixel by,
+
+.. math::
+
+    I = \int\mathrm{d}T\,K(T)\mathrm{EM}(T)\quad[\mathrm{DN}\,\mathrm{pixel}^{-1}\,\mathrm{s}^{-1}],
+
+where :math:`\mathrm{EM}(T)` is the line-of-sight emission measure distribution in a given pixel.
+
+Source Spectra
+--------------
+
+The source spectra describes how a source is emitting as a function of wavelength and temperature.
+It is denoted by :math:`S(\lambda, T)`.
+The source spectra has a physical type of photon per unit time per unit wavelength per solid angle per unit density.
+The units are commonly expressed as
+:math:`\mathrm{photon}\,\mathrm{s}^{-1}\,\mathring{\mathrm{A}}^{-1}\,\mathrm{sr}^{-1}\,\mathrm{cm}^3`.
+The source spectra is specified by the user and can be computed from atomic databases (e.g. CHIANTI).
+This quantity is independent of any instrument properties.
+
+Wavelength Response
+-------------------
+
+The wavelength response describes the instrument sensitivity as a function of wavelength and time.
+The wavelength response is defined as,
+
+.. math::
+
+    R(\lambda,t) = A_{\mathrm{eff}}(\lambda,t)f(\lambda)\frac{p}{gs}\quad[\mathrm{cm}^2\,\mathrm{DN}\,\mathrm{sr}\,\mathrm{photon}^{-1}\,\mathrm{pixel}^{-1}]
+
+It has a physical type of area DN solid angle per photon per pixel.
+
+Camera Gain
+-----------
+
+The camera gain, :math:`g`, describes the conversion between electrons and data number (DN).
+This is a property of the detector.
+The units of the camera gain are as :math:`\mathrm{electron}\,\mathrm{DN}^{-1}`.
+
+Photon-to-Energy Conversion
+---------------------------
+
+The photon-to-energy conversion is given by the amount of energy carried by a photon of wavelength :math:`\lambda`,
+
+.. math::
+
+    f(\lambda) = \frac{hc}{\lambda}\quad[\mathrm{eV}\,\mathrm{photon}^{-1}]
+
+where :math:`h` is Planck's constant and :math:`c` is the speed of light.
+It has a physical type of energy per photon.
+
+Energy-to-Electron Conversion
+-----------------------------
+
+The energy-to-electron conversion, :math:`s`, describes the conversion between electrons released in the detector and the energy of an incoming photon.
+It has a physical type of energy per electron.
+For silicon detectors, a value of :math:`s=3.65\,\mathrm{eV}\,\mathrm{electron}^{-1}` is typically used as this is approximately the energy required to free an electron in silicon.
+
+Pixel Area
```

Comment:
Upon further reflection, I think this needs a more specific name, either "Pixel Solid Angle" or "Pixel Angular Area". "Pixel Area" potentially conflicts with the *physical* area of the pixel, i.e. something with units of cm$^2$ per pixel.